contributors/design-proposals/node/kubelet-cri-logging.md

```diff
@@ -101,7 +101,7 @@ be overlooked. Requirement (2) stems from the fact that disk is managed by
 kubelet as a node-level resource (not per-pod) today, hence it is difficult to
 delegate to the runtime by enforcing per-pod disk quota policy. In addition,
 container disk quota is not well supported yet, and such limitation may not
-even be well-perceived by users. Requirement (1) is crucial to the kubernetes'
+even be well-perceived by users. Requirement (3) is crucial to the kubernetes'
```

Comment:
I think the "Requirement 3: Allow log collectors to easily integrate with Kubernetes across different container runtimes while preserving efficient storage and retrieval" is more crucial to kubernetes' extensibility and and usability across all deployments than Requirement 1.